webservices/args.py

```diff
@@ -181,6 +181,9 @@ def make_seek_args(field=fields.Int, description=None):
     'mur_election_cycles': fields.Int(IStr, required=False, description='Filter MURs by election cycles'),
     'mur_document_category': fields.List(IStr, required=False,
         description='Filter MURs by category of associated documents'),
+    'mur_open_date': fields.Date(required=False, description='Filter MURs by date opened'),
```

Comment:
Apologies for misleading you into thinking that these were only for documentation. They are actually parsed using https://webargs.readthedocs.io/en/latest/quickstart.html#validation.

Also, we need to define all 4 parameters with the right names: `mur_min_open_date`, `mur_max_open_date`, etc.